docs/storage-management.asciidoc

```diff
@@ -3,73 +3,65 @@
 
 [partintro]
 --
-In the following we focus on how you can 
+In the following section you can learn how to
 
 * <<manage-indices-kibana, manage APM indices via Kibana>>
-* <<reduce-storage, reduce storage>> by deleting data 
-or trimming data collection already during ingestion time 
-* <<update-existing-data, update data>> once they are stored 
+* <<reduce-storage, reduce storage usage>>
+* <<update-existing-data, update existing data>>
 --
 
 [[manage-indices-kibana]]
 == Manage Indices via Kibana
-The Kinana UI for managing indices allows you to 
-view your indices, index settings, mappings, docs count, used storage per index and much more. 
-You can perform management operations, 
+The Kibana UI for managing indices allows you to
+view indices, index settings, mappings, docs count, used storage per index and much more.
+You can perform management operations,
 e.g. deleting indices directly via the Kibana UI.
-The UI also supports applying bulk operations on several indices at once. 
-Check out the {kibana-ref}/managing-indices.html[Kibana Managing Indices] docs for how to use the index management UI.
+The UI also supports applying bulk operations on several indices at once.
+Check out the {kibana-ref}/managing-indices.html[Kibana Managing Indices] documentation for more details.
 
 [[reduce-storage]]
 == Reduce storage
-The amount of storage for APM data depends on several factors. 
-Depending on how many services you are instrumenting and how much traffic the services see, 
-the number of recorded transactions varies.
-You can influence the detail level of the information collected 
-by reducing the sample rate or reducing collected stacktrace information.
-Another factor is for how long you want to keep monitoring data around.
+The amount of storage for APM data depends on several factors:
+how many services you are instrumenting, how much traffic the services see, agent and server settings,
+and for how long you keep monitoring data.
 
 [[reduce-sample-rate]]
 [float]
 === Reduce the sample rate
-In case you are monitoring high traffic services you might want to decrease the number of transactions that are sampled. 
-The transaction sample rate directly influences the number of documents to be indexed
-and therefore is the most obvious way to reduce storage. 
 
-The transaction sample rate is controlled in the configuration of agents (for example for {apm-py-ref}/configuration.html#config-transaction-sample-rate[Python] and {apm-node-ref}/agent-api.html#transaction-sample-rate[Node.js]).
+The transaction sample rate directly influences the number of documents (more precisely, Spans) to be indexed
```

Comment:
I don't think "spans" should be capitalized. Typically we capitalize product names, not product terminology or features.

Reply:
right. 
same rule applies eg if it was a link to a section (something like <<spans-section, spans>>?

Reply:
If it's capitalized, I would expect the link text to match the title of the page you're pointing to. Generally, it's better to let asciidoc resolve the link text (unless you're sure that whatever text you provide is stable and won't need to change over time). Lowercase is just fine if the link text is meant to be descriptive.